<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Exp</class>
 <widget class="QDialog" name="Exp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>637</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>720</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#Exp
{
	background:#2D2D2D;
}

</string>
  </property>
  <widget class="QTableView" name="DttableView">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>381</y>
     <width>471</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="ExpB">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>310</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
background-color: rgb(204, 10, 4);</string>
   </property>
   <property name="text">
    <string>ADD</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ExUp">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>310</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
background-color: rgb(204, 10, 4);</string>
   </property>
   <property name="text">
    <string>UPDATE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ExDel">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>310</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
background-color: rgb(204, 10, 4);</string>
   </property>
   <property name="text">
    <string>DELETE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ExLd">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>310</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
background-color: rgb(204, 10, 4);</string>
   </property>
   <property name="text">
    <string>REFRESH</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#label
{
	text-align:left;
	padding-left:26px;
	border:none;	
	color:white;	
	background:#1F1F1F;
}
#label:hover
{
	text-align:left;
	padding-left:26px;
	border:none;	
	color:white;	
	background:#333;
}</string>
   </property>
   <property name="text">
    <string>NAME</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_En">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>50</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#label_2
{
	text-align:left;
	padding-left:26px;
	border:none;	
	color:white;	
	background:#1F1F1F;
}
#label_2:hover
{
	text-align:left;
	padding-left:26px;
	border:none;	
	color:white;	
	background:#333;
}</string>
   </property>
   <property name="text">
    <string>ORGANISATION NAME</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_ExO">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>100</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#label_3
{
	text-align:left;
	padding-left:26px;
	border:none;	
	color:white;	
	background:#1F1F1F;
}
#label_3:hover
{
	text-align:left;
	padding-left:26px;
	border:none;	
	color:white;	
	background:#333;
}</string>
   </property>
   <property name="text">
    <string>DURATION</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Dur">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>150</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>200</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#label_4
{
	text-align:left;
	padding-left:26px;
	border:none;	
	color:white;	
	background:#1F1F1F;
}
#label_4:hover
{
	text-align:left;
	padding-left:26px;
	border:none;	
	color:white;	
	background:#333;
}</string>
   </property>
   <property name="text">
    <string>POSITION</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Pos">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>200</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>250</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#label_5
{
	text-align:left;
	padding-left:26px;
	border:none;	
	color:white;	
	background:#1F1F1F;
}
#label_5:hover
{
	text-align:left;
	padding-left:26px;
	border:none;	
	color:white;	
	background:#333;
}</string>
   </property>
   <property name="text">
    <string>DEPARTMENT</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_ExD">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>250</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="Exconn">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>349</y>
     <width>271</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="HB">
   <property name="geometry">
    <rect>
     <x>614</x>
     <y>0</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
background-color: rgb(255, 176, 107);</string>
   </property>
   <property name="text">
    <string>BACK</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>-1</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#label_6
{
	text-align:left;
	padding-left:26px;
	border:none;	
	color:white;	
	background:#1F1F1F;
}
#label_6:hover
{
	text-align:left;
	padding-left:26px;
	border:none;	
	color:white;	
	background:#333;
}</string>
   </property>
   <property name="text">
    <string>ID</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_id">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>-1</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>100</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-style:italic; color:#00aaff;&quot;&gt;Enter Id to delete and update: &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>380</y>
     <width>21</width>
     <height>221</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#label_7
{
	text-align:left;
	padding-left:26px;
	border:none;	
	color:white;	
	background:#1F1F1F;
}
#label_7:hover
{
	text-align:left;
	padding-left:26px;
	border:none;	
	color:white;	
	background:#333;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
